client/web/src/components/widgets/buttons/PopoverButton.vue

```diff
@@ -63,11 +63,14 @@ export default defineComponent({
 		IconButton,
 	},
 	props: {
+		action: { type: Function, required: false },
```

Comment:
Ah, that's possible. But perhaps there's an argument to be made for consistently using the `{}` (object) syntax and being explicit about which ones are not required? I don't feel strongly about this, but that's maybe something to be addressed in a separate cleanup commit because it probably affects many files.

Reply:
I think the simpler notation is the expected default in Vue, and since it's so widely adopted, I don't see a good reason to increase the verbosity and stricture.

Reply:
Sounds good. I'll include that in a cleanup commit after this and #312 are merged, then.

Reply:
To clarify, I mean the `action: Function` notation as opposed to the object notation. Do you mean that there are instances of `required: false` to be cleaned up?

Reply:
There are 5 locations where we use `required: false`. I'll change those over directly to their type instead of the object notation in the cleanup commit which also adds the blank lines between groups of imports.